methods.md

```diff
@@ -0,0 +1,2 @@
+We rely purely on introspection. 
```

Comment:
Could we say a little bit more what that means? Did we use eyes closed or eyes open, for example?

Reply:
Addressed in commit [9a363f7](https://github.com/arokem/my_better_paper/pull/1/commits/9a363f781eb5b1470fd4eeda1ed55cc4d231cdb3)